extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -229,6 +235,45 @@ public void setPostLogoutPath(Optional<String> postLogoutPath) {
         }
     }
 
+    /**
+     * Default session manager configuration
+     */
+    @ConfigGroup
+    public static class SessionManager {
```

Comment:
Maybe rename this to `CookieManager` ?

Reply:
@pedroigor I'd prefer not to, please see the comment below